src/nested_pandas/series/ext_array.py

```diff
@@ -160,8 +160,16 @@ def to_numpy(self, dtype: None = None, copy: bool = False, na_value: Any = no_de
 
         # Hack with np.empty is the only way to force numpy to create 1-d array of objects
         result = np.empty(shape=array.shape, dtype=object)
+
+        # TODO: delete block once confirmed NA is better to return
+        # fill NaNs - Fill with an empty dictionary using the same columns
+        # nan_vals = pd.isna(array)
+        # if sum(nan_vals) > 0:
+        #    na_idx = np.where(nan_vals)
+        #    array[na_idx] = {key: [] for key in array[np.where(~nan_vals)[0][0]].keys()}
```

Comment:
I like returning NaN values in cases where the dataframe initialization would fail, as it gives us an easy thing at the nestedframe level to filter on to remove empty dataframes. But the annoyance with NaN values is that now all our functions have to handle them in some way, for example `count_nested` applies df.value_counts() to every row, where <NA>.value_counts() will obviously fail.

Reply:
Yes, I would postpone missed value problem for a while. We need to re-implement extension array class first

Reply:
Okay, I'll implement it as is and add a todo